web/dev/sass/modules/_typography.sass

```diff
@@ -7,7 +7,6 @@ $btn-font: 'Droid Sans', sans-serif
 $main-title-font: 'Asap', sans-serif
 $title-font: 'Droid Sans', sans-serif
 $title-font2: 'Asap', sans-serif
-$main-title-font: 'Asap', sans-serif
```

Comment:
please, pay attention, line 7 and line 10 were the same, we don't need duplicate content.